### PR TITLE
Add a how to debug section to the README.md

### PR DESCRIPTION
**Pull request by tobiasqueck**

### CODE REVIEW

**Comment by tobiasqueck** [comment]
@lboehm here is a first draft on how to debug. Is this what you had in mind?

**Comment by lboehm** [comment]
@tobiasqueck Thanks a lot, I will try it out in a few days and I will report back.

**Comment by lboehm** [comment]
Hi @tobiasqueck,

I wanted to fix that issue, but [ran into some problems](https://github.com/SAP/open-ux-tools/issues/214#issuecomment-1003327609) - so I can't say for sure if this guide helps me to get going. But until now I've got one small improvement. You wrote:

> Using the debug terminal, execute all tests with `pnpm test` or a specific one e.g. in `fiori-freestyle-writer`, execute `pnpm test -- test/basic.test.ts`.

I would add the information, that you have to switch directories for running a single test. Maybe something like that:

> Using the debug terminal, execute all tests with `pnpm test` or a specific one e.g. in `fiori-freestyle-writer`, execute `pnpm test -- test/basic.test.ts` in the `fiori-freestyle-writer` directory (`\packages\fiori-freestyle-writer`).

BR
Lukas

**Comment by tobiasqueck** [comment]
@lboehm please check again

File: README.md [hunk 1]
```diff
@@ -82,6 +82,14 @@ To run unit tests using `jest`, run the following command at the root of the rep
 ```shell
 pnpm test
 ```
+**Note**: if the test run fails due to dependency issues, run `pnpm install && pnpm build` in the root of the repository again to make sure all projects are up-to-date.
+
+### Debug packages
+When analyzing a problem, it is helpful to be able to debug the modules. How to debug them depends on the IDE you are using. In this section, it is described how you could debug with VSCode. 
+
+Each of the packages has an extensive set of unit tests covering as many as possible different scenarios, therefore, as a starting point for debugging, it is a good idea to use the tests. The easiest (but not the only) way to debug a specifc test in VSCode is to open a `JavaScript Debug Terminal` and then go to the package that needs to be debugged. Using the debug terminal, execute all tests with `pnpm test` or a specific one, e.g. execute `pnpm test -- test/basic.test.ts` in the `fiori-freestyle-writer` directory (`./packages/fiori-freestyle-writer`). When running either of the commands in the debug terminal, breakpoints set in VSCode will be active.
```
Comment (ullasholla) [inline README.md:90]:
```suggestion
Each of the packages has an extensive set of unit tests covering as many as possible different scenarios, therefore, as a starting point for debugging, it is a good idea to use the tests. The easiest (but not the only) way to debug a specific test in VSCode is to open a `JavaScript Debug Terminal` and then go to the package that needs to be debugged. Using the debug terminal, execute all tests with `pnpm test` or a specific one, e.g. execute `pnpm test -- test/basic.test.ts` in the `fiori-freestyle-writer` directory (`./packages/fiori-freestyle-writer`). When running either of the commands in the debug terminal, breakpoints set in VSCode will be active.
```

**Comment by lboehm** [comment]
@tobiasqueck Looks good! 👍

**Review by unseen1980 — APPROVED** [review]

**Review by ullasholla — APPROVED** [review]
Looks good, thank you.

**Pull request merged**